client/layout/masterbar/style.scss

```diff
@@ -43,7 +43,7 @@ body.is-mobile-app-view {
 	justify-content: space-between;
 	opacity: 1;
 	transition: all 0.3s ease-in-out;
-	--masterbar-item-active-border-radius: 6px; /* stylelint-disable-line scales/radii */
+	--masterbar-item-active-border-radius: 4px;
```

Comment:
I asked @lucasmendes-design on the issue about the radius, he said we had a mix of 4 and 6 in the design and to go with 6 all around on initial implementation.

Reply:
6px feels excessive to me. Lucas. Thoughts on tightening this up just a tad to 4px as the standard?